packages/vite/src/node/server/middlewares/error.ts

```diff
@@ -26,7 +26,11 @@ export function buildErrorMessage(
   includeStack = true
 ): string {
   if (err.plugin) args.push(`  Plugin: ${colors.magenta(err.plugin)}`)
-  if (err.id) args.push(`  File: ${colors.cyan(err.id)}`)
+  let loc = ''
+  if (err.loc) {
+    loc = `:${err.loc.line}:${err.loc.column}`
+  }
```

Comment:
could make this a bit more succinct
```suggestion
  const loc = err.loc ? `:${err.loc.line}:${err.loc.column}` : ''
```

Reply:
I thought the same, but we have both styles in the codebase. But ya, @manucorporat please apply this one and then lets merge. Do you have the allow maintainers to modify the PR check on? I cant seem to be able to merge the suggestion myself.

Reply:
done!